source/sdk/io/native/FileUnix.ooc

```diff
@@ -316,5 +320,17 @@ version (unix || apple) {
 		mkfifo: override func ~withMode (mode: Int32) -> Int {
 			_mkfifo(path as CString, mode as ModeT)
 		}
+
+		createTempFile: static func (pattern, mode: String) -> FStream {
+			result: FStream
+			mask := umask(S_IRUSR | S_IWUSR)
+			fd := mkstemp(pattern)
```

Comment:
Can't find an exact date fpr 2.06, but 2.09 was released in December 1997, do we still care?